instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/internal/HttpRouteState.java

```diff
@@ -24,14 +24,18 @@ public static HttpRouteState fromContextOrNull(Context context) {
     return context.get(KEY);
   }
 
-  public static HttpRouteState create(int updatedBySourceOrder, @Nullable String route) {
-    return new HttpRouteState(updatedBySourceOrder, route);
+  public static HttpRouteState create(
+      @Nullable String method, int updatedBySourceOrder, @Nullable String route) {
```

Comment:
thoughts on changing ordering? this seems more natural to me:

```suggestion
  public static HttpRouteState create(
      @Nullable String method, @Nullable String route, int updatedBySourceOrder) {
```